x-pack/plugins/enterprise_search/public/applications/workplace_search/index.tsx

```diff
@@ -90,24 +89,7 @@ export const WorkplaceSearchConfigured: React.FC<InitialAppData> = (props) => {
         <SourceAdded />
       </Route>
       <Route exact path="/">
-        {errorConnecting ? (
-          <ErrorState />
-        ) : (
-          <Layout navigation={<WorkplaceSearchNav />} restrictWidth readOnlyMode={readOnlyMode}>
-            <Overview />
-          </Layout>
-        )}
-      </Route>
-      <Route path={OAUTH_AUTHORIZE_PATH}>
-        <OAuthAuthorize />
-      </Route>
-      <Route path={SEARCH_AUTHORIZE_PATH}>
-        <SearchAuthorize />
-      </Route>
-      <Route path={PERSONAL_SOURCES_PATH}>
-        <PrivateSourcesLayout restrictWidth readOnlyMode={readOnlyMode}>
-          <SourcesRouter />
-        </PrivateSourcesLayout>
```

Comment:
Curious what happened here? I see your routes got moved below but wondering about the deleted routes

Reply:
Looks like a rebase/merge conflict fix gone bad maybe? My currently open 404 routes PR might cause another one when it lands 😬  LMK if I can help with a merge conflict fix, happy to do so

Reply:
oof, yeah the merge gods have not been kind to me this PR.

Reply:
Actually I'm not seeing this now. Must have been an intermediary state while I was resolving my merge conflict. Although now I have new exciting merge conflicts! 🎉